OracleDatabase/RAC/OracleDNSServer/README.md

```diff
@@ -0,0 +1,67 @@
+# Oracle DNS Server to resolve Oracle RAC IPs
+
+Sample container build files to facilitate installation, configuration, and environment setup for DevOps users.
+
+**IMPORTANT:** This image can be used to setup DNS server for RAC. You can skip this step if you already have a DNS server configure and which can be used for Oracle RAC. You should ensure that the DNS server container is up before starting RAC. This image is provided for test purposes only.
+
+## How to build and run

```

Comment:
This should just be "how to build" because you need to create the bridge before you can run.